plugins/swift/openapi_swift_generator/Sources/openapi_swift_generator/InternalLoader.swift

```diff
@@ -0,0 +1,51 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Stencil
+import Foundation
+
+// A class for loading Stencil templates from compiled-in representations
+
+public class InternalLoader: Loader {
```

Comment:
That would be fine. Here's some [background on that](https://github.com/kylef/Stencil/issues/97). Stencil has an alternate loader called FileSystemLoader that reads from the file system. I think "InternalLoader" would be a good name for use inside Stencil, but see that it's confusing in our code.